maintenance/errors/error_test.go

```diff
@@ -72,3 +76,144 @@ func TestStackTrace(t *testing.T) {
+				"req_id":          "bpboj6bipt34r4teo7g0",
+			},
+			want: &raven.Breadcrumb{
+				Category:  "http",
+				Level:     "debug",
+				Message:   "HTTPS GET www.pace.car",
+				Timestamp: 1582795168,
+				Type:      "http",
+				Data: map[string]interface{}{
+					"method":      "GET",
+					"attempt":     1,
+					"status_code": 200,
+					"duration":    227.717783,
+					"url":         "https://www.pace.car/",
+				},
+			},
+		},
+		{
+			name: "panic_level",
+			data: map[string]interface{}{
+				"level":   "panic",
+				"message": "this is a panic message",
+				"time":    "2020-02-27T10:19:28+01:00",
+			},
+			want: &raven.Breadcrumb{
+				Level:     "fatal",
+				Type:      "error",
+				Message:   "this is a panic message",
+				Timestamp: 1582795168,
+				Data:      map[string]interface{}{},
+			},
+		},
+		{
+			name: "custom_category",
+			data: map[string]interface{}{
+				"level":           "info",
+				"message":         "this is an error message",
+				"sentry:category": "redis",
+				"sentry:type":     "error",
+				"time":            "2020-02-27T10:19:28+01:00",
+				"req_id":          "bpboj6bipt34r4teo7g0",
+			},
+			want: &raven.Breadcrumb{
+				Category:  "redis",
+				Level:     "info",
+				Timestamp: 1582795168,
+				Message:   "this is an error message",
+				Type:      "error",
+				Data:      map[string]interface{}{},
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got, err := createBreadcrumb(tt.data)
+			if (err != nil) != tt.wantErr {
+				t.Errorf("createBreadcrumb() error = %v, wantErr %v", err, tt.wantErr)
+				return
+			}
+
+			assert.Equal(t, tt.want, got, "createBreadcrumb() = %v, want %v", got, tt.want)
+		})
+	}
+}
+
+func Test_Sink(t *testing.T) {
+	rec := httptest.NewRecorder()
+	req := httptest.NewRequest("GET", "/test", nil)
+
+	mux := http.NewServeMux()
+	mux.HandleFunc("/test", func(w http.ResponseWriter, r *http.Request) {
+		require.NotEqual(t, "", log.RequestID(r), "request should have request id")
+
+		client := &http.Client{
+			Transport: transport.NewDefaultTransportChain(),
+		}
+
+		r0, err := http.NewRequest("GET", "https://www.pace.car/", nil)
+		assert.NoError(t, err, `failed creating request to "/succeed"`)
+
+		r0 = r0.WithContext(r.Context())
+		_, err = client.Do(r0)
+		assert.NoError(t, err, `request to "/succeed" should not error`)
+
+		r1, err := http.NewRequest("GET", "http://localhost/fail", nil)
+		assert.NoError(t, err, `failed creating request to "/fail"`)
+
+		r1 = r1.WithContext(r.Context())
+		_, err = client.Do(r1)
+		assert.Error(t, err, `request to "/fail" should error`)
+
+		log.Req(r).Info().
+			Str("sentry:category", "redis").
+			Str("sentry:type", "error").
+			Msg("this is a test message for the sink")
+
+		panic("Sink Test Error, IGNORE")
+	})
+	log.Handler()(Handler()(mux)).ServeHTTP(rec, req)
+
+	resp := rec.Result()
+	defer resp.Body.Close()
+
+	require.Equal(t, 500, resp.StatusCode, "wrong status code")
+}
```

Comment:
What are you testing here? I think at least the name of the test func should give a hint. Because I'm not really seeing what this test is about and what it has to do with the sink.